### PR TITLE
feat: move to workspaces

### PR DESCRIPTION
**Pull request by ririxichan**

### CODE REVIEW

File: packages/marketplace/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+		"outDir": "dist",
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"strict": true,
+		"strictPropertyInitialization": false,
+		"skipLibCheck": true,
+		"noImplicitAny": false,
+		"lib": ["ES2021.String", "dom"]
+	},
+	"include": ["workspaces/marketplace/src/**/*.*"]
```
Comment (CharlieS1103) [inline packages/marketplace/tsconfig.json:16]: Is this correct?

File: packages/marketplace/src/logic/FetchRemotes.ts [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+	// let url = `https://api.github.com/search/repositories?q=${encodeURIComponent("topic:spicetify")}`;
+	if (page) url += `&page=${page}`;
+	// Sorting params (not implemented for Marketplace yet)
+	// if (sortConfig.by.match(/top|controversial/) && sortConfig.time) {
+	//     url += `&t=${sortConfig.time}`
+	const allRepos = await fetch(url)
+		.then((res) => res.json())
+		.catch(() => []);
+	if (!allRepos.items) {
+		Spicetify.showNotification("Too Many Requests, Cool Down.");
```
Comment (CharlieS1103) [inline packages/marketplace/src/logic/FetchRemotes.ts:34]: This is only one example, but it seems as though Spicetify is no longer a valid object?

**Review by CharlieS1103 — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -1,44 +1,19 @@
...
-    "react-dropdown": "^1.10.0",
-    "react-simple-code-editor": "^0.11.2",
-    "semver": "^7.3.7",
-    "spcr-whats-new": "^1.0.1",
-    "spicetify-creator": "^1.0.10",
-    "typescript": "^4.7.3"
-  }
+	"name": "spicetify-marketplace",
+	"version": "0.7.0",
+	"description": "Marketplace for spicetify",
```
Comment (theRealPadster) [inline package.json:4]:
```suggestion
	"description": "Marketplace for Spicetify",
```

File: .gitignore [hunk 4]
```diff
@@ -150,4 +150,4 @@ out
 dist/
 
 # vectornator files for snippet previews
-*.vectornator
+*.vectornator
```
Comment (theRealPadster) [inline .gitignore:153]: Missing EOF newline

**Review by CharlieS1103 — APPROVED** [review]

**Pull request merged**